File: Assets/Plugins/Android/src/com/playphone/multinet/core/ws/data/MNWSSystemGameNetStats.java
//
//  MNWSSystemGameNetStats.java
//  MultiNet client
//
//  Copyright 2011 PlayPhone. All rights reserved.
//

package com.playphone.multinet.core.ws.data;

public class MNWSSystemGameNetStats extends MNWSGenericItem
 {
  public Long getServTotalUsers ()
   {
    return getLongValue("serv_total_users");
   }

  public Long getServTotalGames ()
   {
    return getLongValue("serv_total_games");
   }

  public Long getServOnlineUsers ()
   {
    return getLongValue("serv_online_users");
   }

  public Long getServOnlineRooms ()
   {
    return getLongValue("serv_online_rooms");
   }

  public Long getServOnlineGames ()
   {
    return getLongValue("serv_online_games");
   }

  public Long getGameOnlineUsers ()
   {
    return getLongValue("game_online_users");
   }

  public Long getGameOnlineRooms ()
   {
    return getLongValue("game_online_rooms");
   }
 }

File: Assets/Plugins/iOS/PlayPhoneMultiNet/core/ws/MNWSSystemGameNetStats.m
//
//  MNWSSystemGameNetStats.m
//  MultiNet client
//
//  Copyright 2011 PlayPhone. All rights reserved.
//


#import "MNWSSystemGameNetStats.h"

@implementation MNWSSystemGameNetStats

-(NSNumber*) getServTotalUsers {
    return [self getLongLongValue :@"serv_total_users"];
}

-(NSNumber*) getServTotalGames {
    return [self getLongLongValue :@"serv_total_games"];
}

-(NSNumber*) getServOnlineUsers {
    return [self getLongLongValue :@"serv_online_users"];
}

-(NSNumber*) getServOnlineRooms {
    return [self getLongLongValue :@"serv_online_rooms"];
}

-(NSNumber*) getServOnlineGames {
    return [self getLongLongValue :@"serv_online_games"];
}

-(NSNumber*) getGameOnlineUsers {
    return [self getLongLongValue :@"game_online_users"];
}

-(NSNumber*) getGameOnlineRooms {
    return [self getLongLongValue :@"game_online_rooms"];
}


@end

